private/tools/java/com/github/bazelbuild/rules_jvm_external/resolver/cmd/ConfigArtifact.java

```diff
@@ -0,0 +1,45 @@
+package com.github.bazelbuild.rules_jvm_external.resolver.cmd;
+
+import com.github.bazelbuild.rules_jvm_external.Coordinates;
+import com.google.common.collect.ImmutableSet;
+import java.util.Set;
+
+public class ConfigArtifact {
+
+  // Field names are derived from the values used in `specs.bzl`
+  // Getters are named after the equivalent fields in Maven coordinates
+  private String group;
```

Comment:
Any reason not to name these `groupId` and `artifactId` instead of `group` and `artifact` ?